src/tls.rs

```diff
@@ -463,6 +463,26 @@ impl ServerCertVerifier for NoVerifier {
     }
 }
 
+/// Hyper extension carrying extra TLS layer information.
+/// Made available to clients on responses when `tls_info` is set.
+#[derive(Clone)]
```

Comment:
Should this be `Clone`? Would that be restrictive of adding other types in here that aren't?

Reply:
Hyper requires extensions to be `Clone`.